app/src/lib/git/commit.ts

```diff
@@ -55,7 +55,35 @@ export async function createMergeCommit(
     await unstageAll(repository)
     await stageFiles(repository, files)
     const result = await git(
-      ['commit', '--no-edit'],
+      [
+        'commit',
+        // no-edit here ensures the app does not accidentally invoke the user's editor
+        '--no-edit',
+        // By default Git merge commits do not contain any commentary (which
+        // are lines prefixed with `#`). This works because the Git CLI will
+        // prompt the user to edit the file in `.git/COMMIT_MSG` before
+        // committing, and then it will run `--cleanup=strip`.
+        //
+        // This clashes with our use of `--no-edit` above as Git will now change
+        // it's behaviour to invoke `--cleanup=whitespace` as it did not ask
```

Comment:
Your aussie is showing
```suggestion
        // We should emulate the behavior in this situation because we don't
```